07LessonSeven/homework/index.html

```diff
@@ -4,32 +4,36 @@
   <title>Profession Portal</title>
   <link rel="stylesheet" href="css/style.css">
   </head>
-
   <body>
-    <h1>Profession Portal</h1>
-    <nav>
-      <ul>
-        <li><a target="_blank" href="#">Home</a></li>
-        <li><a target="_blank" href="#">Companies</a></li>
-        <li><a target="_blank" href="#">Positions</a></li>
-      </ul>
-    </nav>
-    <div>
-    <section id="employee">
-      <h4>Hire an Employee</h4>
-      <span><p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultricus facilisis. Ut dapibus blandit nunc, et consectetur dui.</p></span>
-      <form class="sign" action="index.html" method="post">
-        <button class="button" type="button" name="button">Sign up as company</button>
-      </form>
-    </section>
-    <section id="job">
-      <h4>Looking for a job</h4>
-      <span><p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultricus facilisis. Ut dapibus blandit nunc, et consectetur dui.</p></span>
-      <form class="sign" action="index.html" method="post">
```

Comment:
You normally don't need to use the form element here. The form element is used to enclose input elements that will capture user's input. For the button element you can also use <a></a> tags instead. I was expecting for the cursor to change to a hand. Make sure that for every button you create to display the hand cursor use this property in your CSS `cursor: pointer; `  for button's selector.